remote/remote.go

```diff
@@ -31,13 +32,15 @@ type Image struct {
 	image      v1.Image
 	prevLayers []v1.Layer
 	createdAt  time.Time
+	addEmptyLayerOnSave bool
 }
 
 type options struct {
 	platform          imgutil.Platform
 	baseImageRepoName string
 	prevImageRepoName string
 	createdAt         time.Time
+	emptyLayer	  bool
```

Comment:
nit: is it possible we could name this variable `addEmptyLayerOnSave` as well?

Reply:
Done